system/Database/Database.php

```diff
@@ -71,9 +71,9 @@ public function loadForge(ConnectionInterface $db): object
     }
 
     /**
-     * Creates a Utils instance for the current database type.
+     * Creates an Utils instance for the current database type.
```

Comment:
We would say "a Utilities instance" (like "I went to a university") but the shortened version makes both seem weird. What about:
```suggestion
     * Creates an instance of Utils for the current database type.
```